src/rlang/cnd.c

```diff
@@ -100,7 +100,8 @@ void r_interrupt(void) {
 
 enum r_cnd_type r_cnd_type(r_obj* cnd) {
   r_obj* classes = r_class(cnd);
-  if (r_typeof(cnd) != R_TYPE_list ||
+  if (
```

Comment:
Changing this will cause R to crash if the caller assumes (rightfully IMO) that a condition is a list.